proto/lbm/fswap/v1/tx.proto

```diff
@@ -14,7 +14,7 @@ service Msg {
 message MsgSwapRequest {
```

Comment:
`MsgSwapRequest` -> `MsgSwap`.
Also, you may consider specifying target denom in the message.

For `MsgSwapAll`, it would be better to provide from-denom in the message.

Reply:
> Is is to name without `Request` suffix cosmos style? Or any other reason?

I know about the grpc convention, but sadly it's the cosmos style. e.g. `/cosmos.bank.v1beta1.MsgSend`.

> For about proto update, do you mean like this?

Also `from_denom` and `to_denom` in `MsgSwapAll`. It's because the state now has generalized structure, so we need to specify the denoms for the conversion.

Reply:
How about this?
I've added `to_denom` to `MsgSwap` and `from/to_denom` to `MsgSwapAll`.

```protobuf
service Msg {
  rpc Swap(MsgSwap) returns (MsgSwapResponse);
  rpc SwapAll(MsgSwapAll) returns (MsgSwapAllResponse);
}

message MsgSwap {
  // holder's address
  string from_address = 1;
  // from-coin amount
  cosmos.base.v1beta1.Coin from_coin_amount = 2
      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/Finschia/finschia-sdk/types.Coin"];
  string to_denom = 3;
}

message MsgSwapResponse {}

message MsgSwapAll {
  // holder's address
  string from_address = 1;
  string from_denom = 2;
  string to_denom = 3;
}

message MsgSwapAllResponse {}
```